docs-site/docs/development/plan/mqtt-incoming-call-plan.md

```
@@ -0,0 +1,814 @@
+- Create: `.changelog/pr-<PR_NUMBER>.txt` (same convention as `.changelog/pr-2406.txt`)
+
+- [ ] **Step 1: Check existing changelog format**
+
+```bash
+ls .changelog/
+cat .changelog/pr-2406.txt
+```
+
+- [ ] **Step 2: Create file matching the format, e.g.**
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)